### ISSUE TITLE
settings.DATABASES is improperly configured

### ISSUE THREAD

**Issue by freezed** [remace/OC-P13_Archery_Buddy#37]
After a first installation (#35) `manage.py migrate` fails:

```python
(venv) 9:35 user@laptop ~/git/venv % poetry shell
Spawning shell within ~.venvs/venv
9:35 user@laptop ~/git/venv % . ~.venvs/venv/bin/activate
(venv) 9:35 user@laptop ~/git/venv % ./ArcheryBuddy/manage.py makemigrations accounts equipment records 
Migrations for 'accounts':
  ArcheryBuddy/accounts/migrations/0001_initial.py
    - Create model User
Migrations for 'equipment':
  ArcheryBuddy/equipment/migrations/0001_initial.py
    - Create model ArrowRest
    - Create model Barebow
    - Create model Clicker
    - Create model Dampeners
    - Create model Scope
    - Create model CompoundArrowRest
    - Create model CompoundScope
    - Create model Tube
    - Create model Tip
    - Create model Stabilisation
    - Create model Riser
    - Create model OlympicBow
    - Create model Nock
    - Create model Limbs
    - Create model Feathering
    - Create model EquipmentString
    - Create model BergerButton
    - Add field berger_button to barebow
    - Add field limbs to barebow
    - Add field riser to barebow
    - Add field string to barebow
    - Add field user to barebow
    - Create model Arrow
    - Create model CompoundBow
Migrations for 'records':
  ArcheryBuddy/records/migrations/0001_initial.py
    - Create model PracticeRecord
    - Create model StatsRecord
    - Create model StatsRecordSession
    - Add field stats_session to statsrecord
    - Create model PracticeRecordSession
    - Add field practice_session to practicerecord
    - Alter unique_together for practicerecord (1 constraint(s))
    
(venv) 9:35 user@laptop ~/git/venv % ./ArcheryBuddy/manage.py migrate                                  
Traceback (most recent call last):
  File "~git/venv/./ArcheryBuddy/manage.py", line 22, in <module>
    main()
  File "~git/venv/./ArcheryBuddy/manage.py", line 18, in main
    execute_from_command_line(sys.argv)
  File "~.venvs/venv/lib/python3.10/site-packages/django/core/management/__init__.py", line 446, in execute_from_command_line
    utility.execute()
  File "~.venvs/venv/lib/python3.10/site-packages/django/core/management/__init__.py", line 440, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "~.venvs/venv/lib/python3.10/site-packages/django/core/management/base.py", line 402, in run_from_argv
    self.execute(*args, **cmd_options)
  File "~.venvs/venv/lib/python3.10/site-packages/django/core/management/base.py", line 448, in execute
    output = self.handle(*args, **options)
  File "~.venvs/venv/lib/python3.10/site-packages/django/core/management/base.py", line 96, in wrapped
    res = handle_func(*args, **kwargs)
  File "~.venvs/venv/lib/python3.10/site-packages/django/core/management/commands/migrate.py", line 114, in handle
    executor = MigrationExecutor(connection, self.migration_progress_callback)
  File "~.venvs/venv/lib/python3.10/site-packages/django/db/migrations/executor.py", line 18, in __init__
    self.loader = MigrationLoader(self.connection)
  File "~.venvs/venv/lib/python3.10/site-packages/django/db/migrations/loader.py", line 58, in __init__
    self.build_graph()
  File "~.venvs/venv/lib/python3.10/site-packages/django/db/migrations/loader.py", line 235, in build_graph
    self.applied_migrations = recorder.applied_migrations()
  File "~.venvs/venv/lib/python3.10/site-packages/django/db/migrations/recorder.py", line 81, in applied_migrations
    if self.has_table():
  File "~.venvs/venv/lib/python3.10/site-packages/django/db/migrations/recorder.py", line 57, in has_table
    with self.connection.cursor() as cursor:
  File "~.venvs/venv/lib/python3.10/site-packages/django/utils/asyncio.py", line 26, in inner
    return func(*args, **kwargs)
  File "~.venvs/venv/lib/python3.10/site-packages/django/db/backends/base/base.py", line 323, in cursor
    return self._cursor()
  File "~.venvs/venv/lib/python3.10/site-packages/django/db/backends/dummy/base.py", line 20, in complain
    raise ImproperlyConfigured(
django.core.exceptions.ImproperlyConfigured: settings.DATABASES is improperly configured. Please supply the ENGINE value. Check settings documentation for more details.
zsh: exit 1     ./ArcheryBuddy/manage.py migrate
```

**Comment by remace** [comment]
closed by #38

**Comment by freezed** [comment]
Closed with https://github.com/remace/OC-P13_Archery_Buddy/blob/49cc821db6103823a81010ff62adee95adc8bba1/ArcheryBuddy/.env#L1
